src/main/java/org/dita/dost/module/BranchFilterModule.java

```diff
@@ -589,7 +588,7 @@ private void processAttributes(final Element elem, final Branch filter) {
                 final FileInfo dstFileInfo = dstBuilder
                         .build();
 
-                elem.setAttribute(BRANCH_COPY_TO, dstTemp.toString());
+                elem.setAttribute(BRANCH_COPY_TO, dstPathFromMap + FilenameUtils.getName(dstTemp.toString()));
```

Comment:
Not sure about this path and name tweaking, but if it works. Could you add tests for this? Then it would be easier to rewrite this later on if needed.

Reply:
:+1: I'll turn the test case I was using into an integration test.

Reply:
Rather, unit test is preferred. Added that, now cleaning up.